packages/server-core/src/projects/project/project-helper.ts

```diff
@@ -197,19 +197,6 @@ export const getEnginePackageJson = (): ProjectPackageJsonType => {
   return require(path.resolve(appRootPath.path, 'packages/server-core/package.json'))
 }
 
-export const getProjectEnv = async (app: Application, projectName: string) => {
```

Comment:
getProjectEnv is used by the ecommerce project, we need to keep this

Reply:
Fixed, with a comment warning against deleting it.